pkgs/by-name/ph/photocollage/package.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, gettext
+, gdk-pixbuf
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "photocollage";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "adrienverge";
+    repo = "PhotoCollage";
+    rev = "v${version}";
+    hash = "sha256-jDb2mFsok8TNi9+A/FAieqo7YbAUsmrFRBGwdGv71Xg=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    pillow
+    pycairo
+    pygobject3
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+  ];
+
+  nativeBuildInputs = [
+    gettext
+    gobject-introspection
+    wrapGAppsHook
```

Comment:
One more issue I noticed when building locally, is that this input causes the binary to be wrapped twice. See solution in docs:

https://nixos.org/manual/nixpkgs/unstable/#ssec-gnome-common-issues-double-wrapped

Reply:
@doronbehar , should be OK now. How did you notice that the the binary was wrapped twice when building locally?

Reply:
> @doronbehar , should be OK now. How did you notice that the the binary was wrapped twice when building locally?

I pressed <kbd>tab</kbd> for shell completion while I had in my shell line editor `./results/photocollage/bin/`.